### ISSUE TITLE
mir.las.sum: doesn't compile on x86

### ISSUE THREAD

**Issue by wilzbach** [9il/dip80-ndslice#94]
Fails in line 1827, as `nextDown` and `nextUp` aren't defined:

```
assert(nextDown(s) <= r && r <= nextUp(s) || s.isNaN && r.isNaN);
```

**Comment by wilzbach** [comment]
(see also: https://ci.appveyor.com/project/greenify/mir/build/1.0.18/job/9ebrq21u14p1p50e)

**Comment by wilzbach** [comment]
fixed with #95
